osu.Game.Rulesets.Taiko.Tests.iOS/osu.Game.Rulesets.Taiko.Tests.iOS.csproj

```diff
@@ -13,7 +13,8 @@
     </Compile>
   </ItemGroup>
   <ItemGroup Label="Project References">
-    <ProjectReference Include="..\osu.Game\osu.Game.csproj" />
     <ProjectReference Include="..\osu.Game.Rulesets.Taiko\osu.Game.Rulesets.Taiko.csproj" />
+    <ProjectReference Include="..\osu.Game.Tests\osu.Game.Tests.csproj" />
```

Comment:
This causes game tests to show up when running iOS taiko test browser, but it doesn't seem to be easily solvable so I'll just let it be for now, it's not offensive enough for me to care.

Reply:
The other rulesets already do this, I copied this reference from another ruleset's csproj file.

Reply:
I don't see any other test project referencing `osu.Game.Tests`. Usually whenever a test component is used by more than one test project, it is moved to `osu.Game/Tests/...` but let's just keep things as they are for now.